front-end/studio/src/app/pages/apis/{apiId}/editor/_components/forms/shared/media-type-row.component.ts

```diff
@@ -149,4 +150,7 @@ export class MediaTypeRowComponent extends AbstractRowComponent<Oas30MediaType,
         return this.item.schema;
     }
 
+    public deleteAllExamples(mediaType: string): void {
+        let command: ICommand = createDeleteAllExamplesCommand(this.item.ownerDocument(), this.item);
+        this.commandService.emit(command);
```

Comment:
Are you missing an ending `}` here?  The build is failing for this PR for this reason...